### PR TITLE
Add links to petitions waiting for X to homepage

### PR DESCRIPTION
**Pull request by henryhadlow**
It's a bit misleading to only show the petitions that *have* a response or debate outcome

### CODE REVIEW

File: app/helpers/home_helper.rb [hunk 1]
```diff
@@ -18,9 +22,13 @@ def actioned
 
     def generate_actioned
       scope, actioned = Petition.visible, {}
+      awaiting_response = scope.awaiting_response.awaiting_response.by_waiting_for_response_longest
```
Comment (oskarpearson) [inline app/helpers/home_helper.rb]: I think you've got an extra awaiting_response scope here? Not sure if that was intentional.

```
scope.awaiting_response.awaiting_response.by_waiting_for_response_longest
should be
scope.awaiting_response.by_waiting_for_response_longest
```

**Comment by oskarpearson** [comment]
I'm a bit concerned that we're doing a lot more SQL queries on the homepage. I think it might be worth checking the performance impact of this before it hits prod.. Is the home page cached, so that these queries only run occasionally?

There may be indexes that we need to create to make sure that the queries are against indexed fields.

The stats pages don't show a lot of hits of the homepage, so I don't know that it'd be a huge problem... but I think it's worth testing.

**Comment by henryhadlow** [comment]
Over the last week, homepage got 150k pageviews, 3.5% of traffic

**Comment by pixeltrix** [comment]
@oskarpearson @henryhadlow the home page is cached with a five minute expiry and is dependent on the `last_signed_at` column of the `Site` model so the answer is yes, but how effective that caching is could be debated since there are always new signatures happening.

**Comment by henryhadlow** [comment]
@pixeltrix thanks :) can we merge it?

**Comment by pixeltrix** [comment]
@henryhadlow you haven't fixed the double `awaiting_response` scope yet

**Comment by henryhadlow** [comment]
ohhhhh yeahhhhhhh. sorry.

**Pull request merged**